src/languages/en.ts

```diff
@@ -553,7 +553,7 @@ export default {
         payElsewhere: 'Pay elsewhere',
         nextSteps: 'Next Steps',
         requestAmount: ({amount}: RequestAmountParams) => `request ${amount}`,
-        requestedAmount: ({formattedAmount, comment}: RequestedAmountMessageParams) => `requested ${formattedAmount}${comment ? ` for ${comment}` : ''}`,
+        requestedAmount: ({requestor, formattedAmount, comment}: RequestedAmountMessageParams) => `${requestor ? `${requestor} ` : ''}requested ${formattedAmount}${comment ? ` for ${comment}` : ''}`,
```

Comment:
maybe try creating an util function that manage checking a param, smth like 
`getTranslatedString = str => str || ''`